packages/11ty/_includes/components/contributor/bio.js

```diff
@@ -47,11 +47,11 @@ module.exports = function (eleventyConfig) {
 
     const contributorPages = pages.map(({ data, url }) => {
       const { label, subtitle, title } = data
-      return `${link({
+      return `<li>${link({
         classes: ['quire-contributor__page-link'],
         name: pageTitle({ label, subtitle, title }),
         url,
-      })}`
+      })}</li>`
     })
```

Comment:
To keep the list markup as a concern of the method, we might rewrite this as
```
const contributorPagesList = {
  const items = pages.map(({ data, url }) => {
    const { label, subtitle, title } = data
    const classes = ['quire-contributor__page-link']
    const name = pageTitle({ label, subtitle, title })
    return `<li>${link({ classes, name, url })}</li>`
  })
  return html`
    <ul>
      ${items}
    </ul>
  `
}
```
and remove the `<ul>` `</ul>` elements below on lines 67 and 69.

Reply:
@mphstudios, sure, though I'm getting an unexpected identifier error on that code. Seems to be expecting something different in `const contributorPagesList = { ... }`?

```
[11ty] 2. Unexpected identifier (via SyntaxError)
[11ty] Original error stack trace: /Users/galbers/GitHub/quire/packages/11ty/_includes/components/contributor/bio.js:49
[11ty]       const items = pages.map(({ data, url }) => {
[11ty]             ^^^^^
```
One solution might be to leave `contributorPages` as it was, and then add 

```js
const contributorPagesList =
    html`
      <ul>
        ${contributorPages}
     </ul>
   `
```

And then return 

```js
<div class="quire-contributor__details media-content">
  ${contributorImage}
  ${contributorBio}
  ${contributorPagesList}
</div>
```

Reply:
@geealbers I updated/corrected my suggested rewrite of `contributorPages` to use an anonymous function, 

```
const contributorPagesList = () => {
  const items = pages.map(({ data, url }) => {
    const { label, subtitle, title } = data
    const classes = ['quire-contributor__page-link']
    const name = pageTitle({ label, subtitle, title })
    return `<li>${link({ classes, name, url })}</li>`
  })
  return html`
    <ul>
      ${items}
    </ul>
  `
}
```
then
```
<div class="quire-contributor__details media-content">
  ${contributorImage}
  ${contributorBio}
  ${contributorPagesList()}
</div>
```